File: 200709.js
var arr_in = [-7,5,-3,2,-10,3,11];

// var sortedSquares = function(A) {
//     for (let i = 0; i < A.length; i++) {
//         A[i] = A[i] ** 2;
//     }
//     A.sort(function(a, b){return a-b})
//     return A
// };

// console.log(sortedSquares(arr_in))

console.log(arr_in)
arr_in.sort(
    function(a, b){
        return b-a
    })
console.log(arr_in)

File: 200709.py
arr_in = [-7,-3,2,3,11]

class Solution1:
    def sortedSquares(self, A):
        B = [i**2 for i in A]
        return sorted(B)
    

class Solution2:
    def sortedSquares(self, A):
        for i in range(len(A)):
            A[i] = A[i] ** 2
        return sorted(A)

class Solution2:
    def sortedSquares(self, A):
        return [j**2 for j in sorted(abs(i) for i in A)]

j = Solution2()
print(j.sortedSquares(arr_in))